src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int baseLength;
+    private int height;
+
+    public RightTriangle(String color, int height, int bothside) {
```

Comment:
The name of the second parameter 'bothside' is not clear and does not accurately represent the variable. Consider renaming it to 'baseLength' to match the property and improve code readability.